Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs

```diff
@@ -14,21 +15,32 @@
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// A static class encapsulating the Mixed Reality playspace.
+    /// A class encapsulating the Mixed Reality playspace.
     /// </summary>
-    public static class MixedRealityPlayspace
+    [ExecuteAlways]
+    public partial class MixedRealityPlayspace : MonoBehaviour
     {
-        private const string Name = "MixedRealityPlayspace";
+        private const string NameEnabled = "MixedRealityPlayspace";
+        private const string NameDisabled = "MixedRealityPlayspace (Inactive)";
 
-        private static Transform mixedRealityPlayspace;
+        // Cached reference to active playspace object / transform
+        private static MixedRealityPlayspace mixedRealityPlayspace;
+        // Cached reference to camera service
+        private static IMixedRealityCameraSystem cameraService;
 
         public static void Destroy()
         {
-            // Playspace makes main camera dependent on it (see Transform initialization),
-            // so here it needs to restore camera's initial position. 
-            // Without second parameter camera will not move to its original position.
-            CameraCache.Main.transform.SetParent(null, false);
-            UnityEngine.Object.Destroy(mixedRealityPlayspace.gameObject);
+            // This will destroy any main camera parented under this playspace.
```

Comment:
This seems risky. What if I've customized my camera with additional, non-MRTK scripts that the camera service doesn't know about?

Reply:
Since there could be other playspaces loaded when you destroy the active playspace, you don't want to un-parent the camera and leave it hanging. Because another playspace with another main camera may get activated, and then you've got two main cameras to deal with.

It's sort of moot because this destroy method never gets used by any of our services or other scripts. I can revert it if it moves this PR along.